Makefile

```
@@ -210,7 +210,7 @@ libllama.so: llama.o ggml.o $(OBJS)
 # Tests
 #
 
-benchmark: examples/benchmark/benchmark-q4_0-matmult.c ggml.o $(OBJS)
+benchmark: examples/benchmark/benchmark-q4_0-matmult.cpp ggml.o $(OBJS)
```

Since you are already changing the filename, shouldn't we change it to benchmark-matmult together with the Makefile target?

```suggestion
benchmark-matmult: examples/benchmark/benchmark-matmult.cpp ggml.o $(OBJS)
```

(I guess that's obvious, but this is not the only thing that has to be changed in the PR to implement the suggestion)

---

> but doing the other quantization types might be done in `benchmark-matmult.cpp`

Yeah, that was exactly mine logic behind the suggestion.